app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,45 @@
  *= require foundation_and_overrides
 
  */
+
+$margin:10%;
+body {
+ margin: $margin;
+}
+.profile-block-list {
+  text-decoration: none;
+  list-style-type: none;
+}
+@mixin div-blocks($border-color) {
+  border: 3px solid $border-color;
+  padding: 3%;
+  border-radius: 40px;
+}
+@mixin div-hovers {
+  padding: 3%;
+  box-shadow: 5px 5px 5px #bbb;
+}
+.profile-block {
+  @include div-blocks(gray);
+  &:hover {
+    @include div-hovers;
+  }
+}
+.group-block {
+  @include div-blocks(orange);
+  &:hover {
+    @include div-hovers;
+  }
+}
+.task-block {
+  @include div-blocks(purple);
+  &:hover {
+    @include div-hovers;
+  }
+}
+.user-block {
+  @include div-blocks(green);
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.